spot-ml/src/main/scala/org/apache/spot/netflow/FlowWordCreator.scala

```diff
@@ -63,23 +63,13 @@ class FlowWordCreator(timeCuts: Array[Double],
     * @param hour
     * @param minute
     * @param second
-    * @param srcIP
-    * @param dstIP
```

Comment:
Do source IP and destination IP were not used? Or the way to create word changed?